src/main/resources/properties/secure/app-dev.yaml

```diff
@@ -1,10 +1,10 @@
 auth:
   
-  orgId: "![yqKEWZ57M3XUasumfcwhVykLsfUM6yOEv2oEJ04vxZc=]"
+  orgId: "![MZrkNkhH/Z9wpUqTaXJ/tLmHXPLMtMTAs6/5CfaR4A4bVDlv1ye1FkyxVvHGChry]"
   
   # If auth mode = platform-credentials (ignored, otherwise):
-  username: "![Zjv8nFIUwDlayZ6O7DP31p1zQRfrwfjbkmi5DyNiD38=]"
```

Comment:
Let's be careful with these values, seem to be related to a real org, we included fake orgId, username/pass, let's keep the original ones